addons/mrp/models/mrp_routing.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import api, fields, models, _
+from odoo import api, Command, fields, models, _
```

Comment:
```suggestion
from odoo import Command, _, api, fields, models
```
[ASCIIbetically ordered](https://www.odoo.com/documentation/master/contributing/development/coding_guidelines.html#imports)

Reply:
I’ll make sure to remember it. 🧠✅